webpack.prod.config.js

```diff
@@ -3,6 +3,7 @@
 const webpack = require('webpack');
 const webpackConfig = require('./webpack.base.config');
 
+webpackConfig.mode = 'production';
```

Comment:
You can also set the mode with a flag in the npm script (see here https://github.com/dhis2/d2/blob/master/package.json#L8). Webpack will then set `NODE_ENV` for you automatically. But it depends on what you want.

Reply:
yes, there was just so many flags already so I went with adding it to the config since there was different configs available

Reply:
Yeah I saw. That's a bit much. And it's relying on unix tools as well so someone on windows can't even run this.

Reply:
The app is quite old (2 yrs since last commit in master) and something we should really fix moving forward, I agree with that. I think we should at this point get the app out there and put this as a future improvement.

Reply:
> The app is quite old (2 yrs since last commit in master) and something we should really fix moving forward, I agree with that. I think we should at this point get the app out there and put this as a future improvement.

Yeah, sounds good 😊👍

Reply:
I'll create an issue in Jira for the unix specific tooling